chaincfg/genesis.go

```diff
@@ -159,6 +159,77 @@ var testNetGenesisBlock = wire.MsgBlock{
 // test network.
 var testNetGenesisHash = testNetGenesisBlock.BlockHash()
 
+// TestNet2 ------------------------------------------------------------------------
+
+// genesisCoinbaseTx2Legacy is the coinbase transaction for the genesis block for
+// the test network.
+var genesisCoinbaseTx2Legacy = wire.MsgTx{
```

Comment:
I would suggest just reusing the existing `genesisCoinbaseTxLegacy` used by the other testnet network.  These are identical and changing the date in the new genesis block header is sufficient to make it unique.